sentry-ruby/spec/sentry/utils/real_ip_spec.rb

```diff
@@ -13,12 +13,13 @@
     subject do
       Sentry::Utils::RealIp.new(
         :forwarded_for => "192.168.0.2, 2.2.2.2, 3.3.3.3, 4.4.4.4",
+        :forwarded_for => "2.2.2.2, 3.3.3.3, 4.4.4.4, 192.168.0.2",
```

Comment:
why do we need these changes here? I think they have something to do with the `reverse` change, but this line looks like a duplicated entry?

Reply:
I think this ordering more accurately reflects what you would see in a request - the farther right you go in the list the closer you get to the destination. It makes more sense for the proxy ini a private IP address range to be all the way at the end (since it would have been the last proxy in the chain).

Reply:
I see, thanks for explaining 👍